Kernel/System/Console/Command/Maint/Elasticsearch/Migration.pm

```diff
@@ -129,32 +135,32 @@ sub Run {
     if ( $Targets =~ /c/ ) {
         $Self->MigrateCompanies(
             ESObject => $ESObject,
-            Config   => $Config->{Customer} || $Config->{all},
+            Config   => ( ($ConfigIndexSettings && $ConfigIndexSettings->{Customer}) || $Config ),
```

Comment:
I'm a bit torn on this, honestly. I mean on the one hand it's nice, on the other hand, I know that some people will stumble upon this...for now it can stay, I guess, especially as with 10.1 we will sort our the old configuration anyways.

Reply:
Well,... I like compressed code rather than long and verbose blocks of lines reading more like Basic. Sorry if this is really too difficult. I'll try to use a more simple coding in the future.

Reply:
Maybe I should have used the tenary operator?

Reply:
It does look a bit unusual. I probably would have written the check as:
```
my $ConfigIndexSettings = $Kernel::OM->Get('Kernel::Config')->Get('Elasticsearch::IndexSettings') // {};
...
Config   => ($ConfigIndexSettings->{Customer}) // $Config ),
```

Reply:
Ah, yes!  Thank you. I forgot about this operator.